lerna.json

```diff
@@ -3,7 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "2.7.0-dev",
+  "version": "2.7.1",
```

Comment:
The release process should bump the version to `2.8.0-dev`.
Is there a reason why we are not bumping the version to `-dev`?